app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -60,7 +55,7 @@ fun WaterIconText() {
         ),
             contentDescription = "Water Icon",
             modifier = Modifier
-                .size(30.dp)
+                .size(30.dp),
```

Comment:
Added comma as per the kotlin best practices
https://kotlinlang.org/docs/coding-conventions.html